lib/poll.js

```diff
@@ -4,19 +4,33 @@ export default class Poll extends EE.EventEmitter {
   constructor (options, frequency) {
     super();
     this.url = 'https://' + options.host + '/Remote/GetActiveTableID';
-    this.opts = {
-      shortTableId: options.account + '/' + options.table
-    };
+    this.shortTableId = options.account + '/' + options.table
+
+    if (options.cacheBust) {
+      this.cacheBust = options.cacheBust;
+    } else {
+      this.cacheBust = fasle;
```

Comment:
`false` is misspelled (problem with the `eslint` setup?) but also, upon further review, you don't need to this entire if/else block at all. You can replace [lines 9-12](https://github.com/AppGeo/maplarge/pull/14/files#diff-75294b99907e1b8e50b9d3d2e963563eR9-R12) with:

```
this.cacheBust = options.cacheBust;
```

it'll always be either a truthy or falsey value which is fine for the conditional statement below on [line 30](https://github.com/AppGeo/maplarge/pull/14/files#diff-75294b99907e1b8e50b9d3d2e963563eR30)